arrow-optics/src/main/kotlin/arrow/optics/extensions/list.kt

```diff
@@ -24,38 +24,67 @@ import arrow.optics.typeclasses.Index
 import arrow.optics.typeclasses.Snoc
 import arrow.typeclasses.Applicative
 
-fun <A> ListExtensions.traversal(): Traversal<List<A>, A> = ListTraversal()
+@Deprecated(
+  "arrow.optics.extensions package is being deprecated, function is being moved to arrow.optics.traversal",
+  ReplaceWith(
+    "List::class.traversal<A>()",
```

Comment:
I suppose all these need to change based on the new projection over TC companions

Reply:
Yes, I'm working on it and I'll push a commit shortly.